src/Components/Components/src/Routing/RouteEntry.cs

```diff
@@ -25,23 +27,44 @@ public RouteEntry(RouteTemplate template, Type handler, string[] unusedRoutePara
 
         internal void Match(RouteContext context)
```

Comment:
Does https://github.com/dotnet/aspnetcore/blob/master/src/Components/Components/src/Routing/RouteTableFactory.cs#L164-L167 need to be updated to account for optional parameters?

Reply:
Yuppers. Will do.